### ISSUE TITLE
Reduce power usage while lift is idle

### ISSUE THREAD

**Issue by lukasmwerner** [LaSalleRobots/PowerPlay#3]
Proposed change: change the zero power mode of the lift. Current options are float and brake. By setting the lift power to 0 when the lift is at the bottom we can reduce the coil whine and energy use. 

Note: the behavior should change once the lift is at any height above zero with some space for r (error) so enable braking while the lift is up and power is on

**Comment by pdxaeroman** [comment]
Have we done this yet?

**Comment by pdxaeroman** [comment]
Oh wait I remember I tried to and the code didn't work at competition

**Comment by lukasmwerner** [comment]
Yeah we didn't end up using it plus i never pushed my version as I shouldn't be building too much.

**Comment by lukasmwerner** [comment]
This will likely get added on to #5 for an additional feature
